docs/src/main/sphinx/connector/hive.rst

```diff
@@ -459,6 +459,23 @@ with Parquet files performed by the Hive connector.
         definition. The equivalent catalog session property is
         ``parquet_use_column_names``.
       - ``true``
+    * - ``parquet.writer.page-size``
+      - Maximum page size for the Parquet writer.
+      - 1 MB
+    * - ``parquet.writer.block-size``
```

Comment:
I'm a bit confused why Trino uses `block` for the setting prefix and then translates it to Parquet `row group` setting internally. Any ideas?